src/App.js

```diff
@@ -0,0 +1,37 @@
+import React, { useState, useEffect } from 'react';
+import logo from './logo.svg';
+import './App.css';
+import PickupLine from './components/pickupLine';
+
+function App() {
+
+  const [pickupLine, setPickupLine] = useState();
+  const [image, setImage] = useState();
+  const w = window.innerWidth;
+  const h = window.innerHeight;
+
+  useEffect(() => {
+    fetch(`https://picsum.photos/${w}/${h}`)
+    .then(res => res.blob())
+    .then(image => setImage(URL.createObjectURL(image)))
```

Comment:
Nice work, I think that a catch could be added in here on the end for some error handling

Reply:
yes.. I sometimes forget that! thank you!